### ISSUE TITLE
Playback is hard to control

### ISSUE THREAD

**Issue by vladimir-firefly** [MaurycyLiebner/enve#95]
Kudos - I really like this application. It saves me a lot of time in the planning phase of animating traditionally. Cheers!

My first issue is that I find playback control very hard to master.
Play/Pause using space-bar works, but the playback cursor position is ignored. Here's a description of what I mean:

1. Make a long animation (I've even included movie footage and bezier vectors on top of it) ~600 frames long project
2. Press Play
3. Press Stop (at, let's say, frame ~200)
4. Move cursor backward by sliding vertical line on the ruler with a mouse (to the frame 100)
5. Press Play (Space again)
6. The animation plays from frame 200, not 100, where that vertical playback cursor still resides. 

I would expect that the new replay starting point is now 100, not ~200.

Is this a design decision or a bug?

Regards,
Vladimir

**Comment by MaurycyLiebner** [comment]
I think what you are referring to as Stop is actually Pause.
If you want to change the playback starting frame you have to first stop the current preview (press ESC on the keyboard or trigger the Stop button), and then move to a different frame and start a new preview (press space-bar or trigger the Play button).

Does it resolve your issue?

**Comment by vladimir-firefly** [comment]
Now I understand how it's designed!
But wouldn't it be better to stop preview automatically on the arrow left/right and direct positioning of the cursor with the mouse? It is obvious that the user wanted to do something when moving the cursor position. It is now confusing to me that I got to some position, viewed what there is to see at that frame and then upon hitting space I got some kind of a jump. A rather strange mental jump also.

**Comment by MaurycyLiebner** [comment]
Yes, it would probably make sense to cancel preview on some user interactions.
